andela/lotus-ah-frontend
# #162080971 User can view profile page

#### What does this PR do?
Implement view user profile page

#### Description of Task to be completed?
- Fixed the header and the notification icon
- A user can view their profile page

#### How should this be manually tested?
Anyone with the right url can view any user profile. From localhost;
1. After cloning or pulling the branch to your local
2. Run yarn install to install all necessary packages and run yarn start:dev start your app on port 5000
3. visit the right url e.g `http://localhost:8080/profile/Prince_41` to see a user profile.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
-[#162080971](https://www.pivotaltracker.com/story/show/162080971)
#### Screenshots (if appropriate)
<img width="1680" alt="screenshot 2018-11-21 at 12 39 33 pm" src="https://user-images.githubusercontent.com/14542449/48839076-92dc9700-ed8a-11e8-9ad9-f35ecc8f9bf8.png">

#### Questions:
N/A
